tests/linux/lodsb.c

```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
+
+static inline char *mystrcpy(char *dest, const char *src)
+{
+    int d0, d1, d2;
+    __asm__ __volatile__(  "1:\n\t"
+            "\tlodsb\n\t"
```

Comment:
Get rid of these `\t`s and `\n`s and use `;`s to separate the instructions. E.g. ```"lodsb;"
"stosb;"
...```

`\n` and `\t` just clutter things and are only meaningful if you want to actually inspect the assembly before it is assembled.